src/sentry/data_export/models.py

```diff
@@ -33,7 +33,7 @@ class ExportedData(Model):
     __include_in_export__ = False
 
     organization = FlexibleForeignKey("sentry.Organization")
-    user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="SET_NULL")
+    user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete=models.SET_NULL)
```

Comment:
This seems dangerous.  This behavior does not match at all.
For instance, `models.SET_NULL` is a synchronous behavior that maps to a transaction completing.  However, "SET_NULL" is a HybridCloudForeignKey specific behavior that does not map to any of the `models.SET_NULL` guarantees.  I worry this is highly misleading.  I'm willing to change the name of the values, but I think it's dangerous to share the enum.

Reply:
django 3.x does not allow this to be **stringly-typed** -- what would you suggest

Reply:
Understood.  I'll take a stab at the PR.

I'm hoping that we can use a different name while retaining the same typing.  This way we can separate semantics, play cool with typing, and avoid a database state issue.